### PR TITLE
NIAID CT real study viewer

### PR DESCRIPTION
**Pull request by paulineribeyre**
Link to Jira ticket if there is one:

### Environments
NIAID CT

### Description of changes
portal to nct-1.0.0 and real study viewer config

### CODE REVIEW

**Review by atharvar28 — COMMENTED** [review]
lgtm 👍

File: accessclinicaldata.niaid.nih.gov/manifest.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "pidgin": "quay.io/cdis/pidgin:2020.08",
     "revproxy": "quay.io/cdis/nginx:2020.08",
     "sheepdog": "quay.io/cdis/sheepdog:2020.08",
-    "portal": "quay.io/cdis/data-portal:feat_wireframe-poc",
+    "portal": "quay.io/cdis/data-portal:ct-1.0.0",
```
Comment (mfshao) [inline accessclinicaldata.niaid.nih.gov/manifest.json:19]:
```suggestion
    "portal": "quay.io/cdis/data-portal:nct-1.0.0",
```

**Review by mfshao — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# accessclinicaldata.niaid.nih.gov/manifest.json
## :warning: Services on branch
- portal

**Comment by PlanXCyborg** [comment]
# accessclinicaldata.niaid.nih.gov/manifest.json
## :warning: Services on branch
- portal

**Pull request merged**